src/DripsHub.sol

```diff
@@ -172,10 +172,10 @@ abstract contract DripsHub {
         int128 nextCycle;
     }
 
-    struct SenderId {
-        bool isSubSender;
-        address senderAddr;
-        uint256 subSenderId;
+    struct UserOrAccount {
```

Comment:
I would prefer a generic term here.

Sender
  - User
  - Account
  
Maybe Sender or something else which can be either a user or an account?

Reply:
It's going to be hard to get any more generic :D `UserOrAccount` is clunky, but it doesn't limit its usage and clearly reuses existing terms. `Sender` would reintroduce a new term (we're dropping it) and it would make `_give(Sender memory sender, ...)` signature a bit weird.